### PR TITLE
Using brightness in analog clock overlay

### PR DESCRIPTION
**Pull request by paspiz85**
I want to adjust the brightness of the analog clock. I used the existing brightness setting to decrease the intensity of the 3 LEDs related to the analog clock overlay.
I tested on my ESP8266 Wemos D1 mini

### CODE REVIEW

**Comment by blazoncek** [comment]
That's not how it should go.
Use `color_fade()` available in 0_15 branch.

File: wled00/overlay.cpp [hunk 1]
```diff
@@ -11,6 +11,7 @@ void _overlayAnalogClock()
   {
     _overlayAnalogCountdown(); return;
   }
+  uint8_t brightness = strip.getBrightness();
```
Comment (softhack007) [inline wled00/overlay.cpp:14]: @blazoncek not sure what is better to retrieve the global brightness - simply use global `bri`, or use `‎WS2812FX::getBrightness‎()`?

Comment (blazoncek) [reply wled00/overlay.cpp:14]: Yes. That would be in line with other code that uses it.

Comment (paspiz85) [reply wled00/overlay.cpp]: ok

**Pull request merged**